identity-and-access-management/secret-manager/api-cli/create-secret-with-type.mdx

```diff
@@ -0,0 +1,72 @@
+---
+meta:
+title: Creating a secret with a type using the Secret Manager API
+description: This page explains how to create a secret with a type using the API.
+content:
+h1: Creating a secret with a type using the Secret Manager API
+paragraph: This page explains how to create a secret with a type using the API.
+tags: secret-management type secret-manager api create
+categories:
+  - identity-and-access-management
+---
+
+This documentation allows you to create a secret with a type and to create a version matching this type using the API.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+
+1. Create a secret with a `database_credentials` type from the API by launching the following command:
+```
+    curl -q \
+      -H "X-Auth-Token: $SECRET_KEY" \
+      -H 'Content-Type: application/json' \
+      -X POST \
+      -d '{
+            "name": "my-secret",
+            "project_id": "'$SCW_PROJECT_ID'",
+            "type": "database_credentials"
+      }' \
+      https://api.scaleway.com/secret-manager/v1beta1/regions/fr-par/secrets
+```
+
+2. Encode the content of your secret version in bytes :
+```
+{
+    "engine": "mysql",
+    "username": "username",
+    "password": "my-super-secure-password",
+    "host": "<instance host name/resolvable DNS name>"
```

Comment:
it can also be an IP address

Reply:
also it's missing a semicolon at the end of the line

Reply:
Correct, thank you !